### PR TITLE
add testing for negative precision

### LINKED ISSUE #69: Additional tests for round math util
Improve testing to include negative values.

### PR DESCRIPTION
**Pull request by kalisjoshua**
closes #69

### CODE REVIEW

File: packages/math/src/round/index.test.ts [hunk 1]
```diff
@@ -13,22 +13,41 @@
...
-  it('rounds to precision', () => {
-    const a = round(1, value);
-    const b = round(2, value);
-    const c = round(3, value);
-    const d = round(4, value);
+  describe('positive rounding; decimal position', () => {
+    const value = 1.2345;
+
+    it.each`
+      precision | expected
```
Comment (brandonjpierce) [inline packages/math/src/round/index.test.ts:21]: Neat TIL!

**Review by brandonjpierce — APPROVED** [review]
